Library/Formula/nss.rb

```diff
@@ -22,8 +28,8 @@ def install
     args = [
       "BUILD_OPT=1",
       "NSS_USE_SYSTEM_SQLITE=1",
-      "NSPR_INCLUDE_DIR=#{HOMEBREW_PREFIX}/include/nspr",
-      "NSPR_LIB_DIR=#{HOMEBREW_PREFIX}/lib"
+      "NSPR_INCLUDE_DIR=#{Formula["nspr"].include}/nspr",
+      "NSPR_LIB_DIR=#{Formula["nspr"].lib}"
```

Comment:
Side note — it would be awesome if [Formula Cookbook](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#variables-for-directory-locations) mentioned `opt_include`, `opt_lib`, and other possible `opt_`-fields.